fairscale/nn/pipe/async_pipe.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All rights reserved.
+#
+# This source code is licensed under the BSD license found in the
+# LICENSE file in the root directory of this source tree.
+
+from .multiprocess_pipe import MultiProcessPipe
+from .types import PipelineStyle
+
+
+class AsyncPipe(MultiProcessPipe):
```

Comment:
add a class docstring?

Also, curious, why type: ignore is needed for its init?

Reply:
I think because I'm not directly handling the args.